Classes/Service/MauticAuthorizeService.php

```diff
@@ -265,4 +266,79 @@ protected function translate(string $key): string
         }
         return $this->languageService->sL('LLL:EXT:mautic/Resources/Private/Language/locallang_mod.xlf:' . $key);
     }
+
+    public function validateAccessToken(): bool
+    {
+        if ($this->extensionConfiguration['authorizeMode'] === YamlConfiguration::OAUTH1_AUTHORIZATION_MODE) {
+            if ($this->extensionConfiguration['accessToken'] !== ''
```

Comment:
This can be simplified:
`return $this->extensionConfiguration['accessToken'] !== '' && $this->extensionConfiguration['accessTokenSecret'] !== ''`